popup1.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>

```

Comment:
There is actually some profound truth in the old quip that there are two hard problems in CS: cache invalidation and naming things (I'm talking about the naming). It forces you to think about and understand what something is for -- whether that's a variable, function, or file. It's a very useful exercise. Try to get into the habit of giving everything a meaningful name. It's very important when you're working with others (which all real software projects will be), and you look silly in an interview if you name things "thing", "thing1", etc.